src/Docker.DotNet/Models/Limit.Generated.cs

```diff
@@ -0,0 +1,17 @@
+using System.Runtime.Serialization;
+
+namespace Docker.DotNet.Models
+{
+    [DataContract]
+    public class Limit // (swarm.Limit)
```

Comment:
All of the Swarm stuff we have called "SwarmLimit", "SwarmJobStatus" perviously since these are not namespaced to avoid name collisions

Reply:
Should be renamed in next commit